test_vectordraw.py

```diff
@@ -183,6 +183,22 @@ def test_check_segment_angle(self):
         self.assertFails(vd.check_segment_angle(self.check(-45, 9, errmsg=custom_errmsg), vectors),
                          custom_errmsg)
 
+    def test_check_points_on_line(self):
+        errmsg = 'The line vec does not pass through the correct points.'
+        vectors = {'vec': self.vector(1, 1, 5, 5)}
+        self.assertPasses(vd.check_points_on_line(self.check([[1, 1], [5, 5]]), vectors))
+        self.assertPasses(vd.check_points_on_line(self.check([[1, 2], [5, 4]]), vectors))
+        self.assertPasses(vd.check_points_on_line(self.check([[3.7, 2.3], [2.3, 3.7]]), vectors))
```

Comment:
@smarnach nit: I would argue that (1,2) lies on (1,1) -> (5,5) line. Default tolerance is way too big here. So I would suggest passing lower tolerance to `self.check`. Feel free to ignore it, but I understood that tolerance in play only by looking at the rest of the code.